lib/MqttClient.js

```diff
@@ -384,7 +384,7 @@ class MqttClient {
                  */
                 case CUSTOM_COMMANDS.ZONED_CLEANUP:
                     if (Array.isArray(msg.zone_ids) && msg.zone_ids.length) {
-                        this.vacuum.startCleaningZoneByName(msg.zone_ids)
+                        this.vacuum.startCleaningZonesById(msg.zone_ids)
```

Comment:
This broke my node-red interface.
MQTT should use the zone names.
I don't want to use the ids here.

Reply:
Fair. Done.